app/Http/Controllers/AdminDash.php

```diff
@@ -1381,30 +1382,27 @@ public function removePosition($id) {
         return redirect()->back()->with('success', 'The position has been removed successfully.');
     }
 
-    public function toggleRegistration($id) {
+    public function toggleRegistration($id): RedirectResponse {
         $event = Event::find($id);
 
-        if ($event->reg == 0) {
-            $event->reg = 1;
-            $event->save();
+        if (! $event) {
+            return redirect()->back()->with('error', 'That event does not exist');
+        }
 
-            $audit = new Audit;
-            $audit->cid = Auth::id();
-            $audit->ip = $_SERVER['REMOTE_ADDR'];
-            $audit->what = Auth::user()->full_name.' opened registration for the event '.$event->name.'.';
-            $audit->save();
-        } elseif ($event->reg == 1) {
-            $event->reg = 0;
-            $event->save();
+        $event->toggleRegistration();
 
-            $audit = new Audit;
-            $audit->cid = Auth::id();
-            $audit->ip = $_SERVER['REMOTE_ADDR'];
-            $audit->what = Auth::user()->full_name.' closed registration for the event '.$event->name.'.';
-            $audit->save();
+        return redirect()->back()->with('success', 'The registration has been toggled successfully.');
+    }
+
+    public function toggleShowAssignments($id): RedirectResponse {
+        $event = Event::find($id);
+        if (! $event) {
+            return redirect()->back()->with('error', 'That event does not exist');
         }
 
-        return redirect('/dashboard/controllers/events/view/'.$id)->with('success', 'The registration has been toggle successfully.');
```

Comment:
Change to read 'The registration has been toggled successfully.'

Reply:
This was a fix I made. See line 1394 above for the added line (this one was removed). The diff is just showing things a little mixed